docs/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <title>Clever Design System</title>
-    <link rel="shortcut icon" href="favicon.ico?1" >
+    <link rel="shortcut icon" href="favicon.ico" >
```

Comment:
Yeah, it's basically just a cache buster. Favicons are cached by browsers so adding a different query param allows us to prompt the browser to re-download. You can just increment the count if you're updating the icon